test/specs/components/RadioGroup/RadioGroup-test.tsx

```diff
@@ -101,31 +101,33 @@ describe('RadioGroup', () => {
       })
     })
 
-    describe('click event handler', () => {
-      it('should set the value when item is clicked', () => {
-        const checkedValueChanged = jest.fn()
-        const wrapper = mountWithProvider(
-          <RadioGroup items={getItems()} checkedValueChanged={checkedValueChanged} />,
-        )
-        const radioGroupItems = wrapper.find('RadioGroupItem')
+    if (isShorthandApiTest) {
+      describe('click event handler', () => {
+        it('should set the value when item is clicked', () => {
```

Comment:
"should set `checked` when item is clicked"? At least this is what I see from test